pygsti/objects/localnoisemodel.py

```diff
@@ -717,6 +717,8 @@ def __init__(self, num_qubits, gatedict, prep_layers=None, povm_layers=None, ava
         #SPAM (same as for cloud noise model)
         if prep_layers is None:
             pass  # no prep layers
+        elif isinstance(prep_layers, _sv.SPAMVec): # just a single SPAMVec
```

Comment:
Does the docstring state that giving a single spamvec is allowed behavior?  If not, we should add it.

Reply:
It actually already was but was vague as stated. I modified the docstring to include the allowed types (SPAMVec or POVM).

Note that technically, LocalNoiseModel can still take a single LinearOperator as a prep. I didn't change this behavior, although CHP no longer uses this code and now takes SPAMVecs.